lms/djangoapps/email_marketing/signals.py

```diff
@@ -191,3 +200,16 @@ def _get_current_site():
         return
 
     return {'id': request.site.id, 'domain': request.site.domain, 'name': request.site.name}
+
+def _log_sailthru_api_call_time(time_before_call):
+    """
+    Logs Sailthru api synchronous call time
+    """
+
+    time_after_call = datetime.datetime.now()
+    delta_sailthru_api_call_time = time_after_call - time_before_call
+
+    log.info("Sailthru api call started at %s and ends at %s, time spent:%s milliseconds",
+             time_before_call.isoformat(' '),
+             time_after_call.isoformat(' '),
+             delta_sailthru_api_call_time.microseconds()/1000)
```

Comment:
Are you sure `.microseconds` is a function, not a property/attribute of the object? See doc https://docs.python.org/3.5/library/datetime.html#timedelta-objects